src/expr/impl/src/udf/wasm.rs

```diff
@@ -92,6 +92,17 @@ fn create_rust(opts: CreateFunctionOptions<'_>) -> Result<CreateFunctionOutput>
 
     let wasm_binary = std::thread::spawn(move || {
         let mut opts = arrow_udf_wasm::build::BuildOpts::default();
+        // Use the same chrono feature setting as src/expr/impl/Cargo.toml
+        // Use a fixed chrono version 0.4.39 because the latest 0.4.40 failed to compile.
+        // TODO: may avoid setting a fixed version when succeed to compile with latest chrono version
```

Comment:
🤔 I would rather we pin the dependencies to specific versions (with `Cargo.lock`) and document them to get more reproducible results.

Reply:
Agree. Maybe we can first support specifying `Cargo.lock` file used to build wasm in `arrow_udf_wasm`, and then we can include workspace `Cargo.lock` using `include_str`. For e2e_test, shall we let it reuse the workspace `Cargo.lock` with soft link to the lock file?